test/e2e/persistent_volumes-disruptive.go

```diff
@@ -159,7 +159,7 @@ func testKubeletRestartsAndRestoresMount(c clientset.Interface, f *framework.Fra
 	By("Testing that written file is accessible.")
 	_, err = podExec(clientPod, "cat "+file)
 	Expect(err).NotTo(HaveOccurred())
-	framework.Logf("Pod %s detected %s after kubelet restart", clientPod.Name, file)
+	framework.Logf("Volume mount detected on pod and written file %s is readable post-restart.", file)
```

Comment:
L128 `testItStmt: "Should test that a file written to the mount before kubelet restart is stat-able after restart."`:
Why not:  "Should test that a file written to the mount before kubelet restart can be read after restart." ?
_stat_ is not directly called by the test, though it is called by linux read/write implementations.